src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -47,14 +47,20 @@
     private boolean                    pseudo;
     private boolean                    collectSource;
     private boolean                    cleanCache;
-    private long                       cacheExpiredTime;
+    
+    @Deprecated
```

Comment:
It's not good to add '@Deprecated' fields or functions in the new codes.
How about this judge: if cacheExpiredTime is set, use it to do expiration check and ignore max-age; else use max-age to check?